libraries/datahandler/src/main/java/org/eclipse/sw360/datahandler/permissions/PermissionUtils.java

```diff
@@ -174,4 +174,44 @@ public static <T> DocumentPermissions<T> makePermission(T document, User user) {
         }
     }
 
+    public static Set<String> editableParmsSet = new HashSet<>();
+    static {
+        editableParmsSet.add("enableSvm");
+        editableParmsSet.add("enableVulnerabilitiesDisplay");
+        editableParmsSet.add("projectManager");
+        editableParmsSet.add("projectOwner");
+        editableParmsSet.add("securityResponsibles");
+        editableParmsSet.add("externalIds");
+        editableParmsSet.add("state");
+        editableParmsSet.add("phaseOutSince");
+    }
```

Comment:
Please merge the variable and static block with existing variables (at the beginning of the class) and the static block.